### ISSUE TITLE
Backup fails when the blocked jobs list includes company names with non-ASCII characters

### ISSUE THREAD

**Issue by Matej9937** [damianmgarcia/Hide-n-Seek#34]
The Backup button on the Brave browser didn't perform a backup after clicking, probably because it is not a supported browser. Although Brave uses Chromium as a base and shares many similarities with Chrome, I guess this shouldn't be a hassle to fix.

I was able to locate the file whose content when replaced restored the 400+ saved companies for blocking.

**Folder name / Pathway**
C:\Users\Matej\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default\Local Extension Settings\ajneldpbecoimjjcbmnhkinmhagejnnl
Note: "Matej" is the username, and "ajneldpbecoimjjcbmnhkinmhagejnnl" is the default extension name; these might vary for different users.

**Folder content:**
![image](https://github.com/user-attachments/assets/35b28af6-e606-4801-ad13-22b235f82775)

**File name:** 
000003.log

If you need more data or any help, let me know. I would be happy to help.

PS: This is my first issue submission on GitHub, I hope I did everything correctly. :)

**Comment by damianmgarcia** [comment]
Hi Matej! The notification message sent by GitHub when you created this issue 2 weeks ago was sent to my spam folder, so that's why my response is so late. Sorry about that!

I haven't been able to reproduce this issue yet. Here is what I've tried:

1. I downloaded Brave, then installed Hide n' Seek from the Chrome Web Store, then blocked a few jobs, and then clicked the "Backup" button. This opened the file picker and allowed me to save the backup to my device. 
2. I removed the extension from the previous step and then installed it locally using the "Load unpacked" feature within brave://extensions/. However, the backup function still worked.

Do you experience this issue even if you install it from the Chrome Web Store? I'm assuming that yours is not installed via the Chrome Web Store because the extension id in your extension settings is ajneldpbecoimjjcbmnhkinmhagejnnl, which doesn't match the id of Hide n' Seek when supplied by the Chrome Web Store, which is agghbaheofcoecndkbflbnggdjcmiaml.

**Comment by Matej9937** [comment]
Hi Damian, no worries :).

Surprisingly I was able to reproduce your step 1. as well as 2. 

After several hours of testing different .log and .json files and official and unofficial builds I discovered the core of the issue:

If company name within the .json file (the official backup file from official addon from Chrome Web Store) contains one or more from the following 3 symbols the backup button stops working.
1. –
2. •
3. 🧭
Example:
![image](https://github.com/user-attachments/assets/6cd7720e-ba87-46ea-9f68-752fc30f8379)

So I guess that my 000003.log contains the symbols which somehow break the backup as is in the case of .json file.

Conclusion: Brave backup works, sorry for wrong call, although companies with "–", "•" or "🧭" break the backup functionality.
